tapioca/adapters.py

```diff
@@ -90,6 +95,9 @@ def format_data_to_request(self, data):
     def response_to_native(self, response):
         return {'text': response.text}
 
+class TokenHTTPRefresherMixin(object):
```

Comment:
Maybe we should make a ``Oauth2TokenRefreshMixin``, so it's ready to go example of a mixin. It should expect the user to have passed ``client_id``, ``client_secret``, and a ``access_token`` in the intialization of the wrapper. Here is a an example of how it works:
http://requests-oauthlib.readthedocs.org/en/latest/examples/real_world_example_with_refresh.html

If you prefer, we can remove ``TokenHTTPRefresherMixin`` for now, approve this PR and you can make another one for the ``Oauth2TokenRefreshMixin``, so we have more time to test it and play a little with it before merging it.